webapi/src/test/scala/org/knora/webapi/responders/admin/ProjectsServiceLiveSpec.scala

```diff
@@ -263,4 +240,65 @@ object ProjectsServiceLiveSpec extends ZIOSpecDefault {
       _ <- projectsService.provide(actorToZioBridge)
     } yield assertTrue(true)
   }
+
+  val getProjectMembers = suite("get all members of a project")(
+    test("get members by project shortcode") {
+      val shortcode      = "0001"
+      val identifier     = TestDataFactory.projectShortcodeIdentifier(shortcode)
+      val requestingUser = KnoraSystemInstances.Users.SystemUser
+      val projectsService =
+        ZIO
+          .serviceWithZIO[ProjectsService](_.getProjectMembers(identifier, requestingUser))
+          .provideSome[ActorToZioBridge](layers)
+      val actorToZioBridge = ActorToZioBridgeMock.AskAppActor
+        .of[ProjectMembersGetResponseADM]
+        .apply(
+          assertion = Assertion.equalTo(ProjectMembersGetRequestADM(identifier, requestingUser)),
+          result = Expectation.value(ProjectMembersGetResponseADM(Seq.empty[UserADM]))
+        )
+        .toLayer
+      for {
+        _ <- projectsService.provide(actorToZioBridge)
+      } yield assertTrue(true)
```

Comment:
AFAIK there is `zio.test.package$#assertCompletes` as an alias. Rhat probably makes more sense when reading tests that merely assert the completion of something.

```suggestion
      } yield assertCompletes
```

Reply:
much nicer, thanks! :)